### DISCUSSION TITLE
Incorrect size when entered in "cm"

### DISCUSSION THREAD

**Discussion by mahxds** [Budibase/budibase#8659] [Help]
Hi ,

iam experiencing some weird effect currently.

Since there aint a PDF modul, iam trying to make printable view.

For this i tried to fix the screen size in cm, wasnt working 
I tried to put a container as a sizing modul and set ist to fixed cm as well. Didnt work.

No matter what i tried, the printout was wrong (to small) and didnt do what i entered in the cm.
And the weird part is, that its off by like 30%. In height and width.

I know i can just compensate this by entering bigger values, but thats just a termporary fix.

Has anyone experienced same ?

**Comment by melohagan** [comment]
Hey @mahxds 

Could you share some screenshots please?
